frontend/src/app/components/MainInstallButton.js

```diff
@@ -1,3 +1,5 @@
+// @flow
```

Comment:
Shouldn't this file live in a directory `frontend/src/app/components/MainInstallButton/index.js`?

Reply:
Eventually, yeah, but I didn't want to spread out into too many changes while focusing on ExperimentPage